bleachbit/FileUtilities.py

```diff
@@ -924,12 +954,26 @@ def estimate_completion():
                 break
             else:
                 raise
+
+        # Get the file system type from the given path
+        fstype = get_filesystem_type(pathname)
+        fstype = fstype[0]
+        print('File System:' + fstype)
```

Comment:
Please replace `print()` function with `logging.debug()`

Reply:
Ok. I will replace it.